_i18n/es.yml

```diff
@@ -140,6 +140,47 @@ publications-tweet_feed:
 publications-post:
    author_title: Por
    date_title: Publicado
+services-software_creation:
+   title: Creación de software
+   subtitle: <p><strong>Te ayudaremos a crear software bien elaborado que aporta valor a su negocio.</strong></p>
+           <p>Nuestros craftsmen han perfeccionado su destreza en tecnologías y prácticas para poder entregar a tiempo y con calidad. Nos ajustamos a sus necesidades.</p>
```

Comment:
@alvarogarcia7 I also wondered but I think it will be better understood than using something like 'artesano'